docs/linter/eslint-plugin/rules/no-inner-html.md

```diff
@@ -0,0 +1,20 @@
+# no-inner-html
+
+This rule ensures that your template does not use innerHTML.
```

Comment:
Could be good to indicate the reason/motivation of it.
Having an innerHtml in the template is it a bad practice? Is it something we do want to encourage to all the application (or only for RefX or Application having CMS capabilities)?

If this is only for certain application (or in certain condition) maybe it should be off per default (and be activated on demand)